app.py

```diff
@@ -39,6 +40,58 @@ def mirror(name):
     return create_response(data)
 
 # TODO: Implement the rest of the API here!
+@app.route('/users')
+def get_all_users():
+    team_name = request.args.get('team')
+    if not team_name:
+        return create_response(db.get('users'))
```

Comment:
`create_response` should take in a dictionary (see the docstring in `create_response`). Note that `db.get` returns a list of dictionaries